R/bootstrap-model.R

```diff
@@ -0,0 +1,681 @@
+#' Create a boostrap model object from an existing model
+#'
+#' @param .mod a `bbr` model object
+#' @param .suffix a suffix for the bootstrap run directory. Will be prefixed by
+#'  the model id of `.mod`.
+#' @inheritParams copy_model_from
```

Comment:
We need to define `.inherit_tags` here too, because we inherit from `copy_model_from()`, which says "If FALSE, the default, new model will..."

That is correct for `copy_model_from()` but for `new_bootstrap_run()` this defaults to `TRUE`.

Related: I do like this defaulting to `TRUE`, even though `copy_model_from()` doesn't. We just need the docs to reflect that.